src/lightmap.cpp

```diff
@@ -1237,8 +1237,12 @@ float fastexp( float x )
         float f;
         int i;
     } u, v;
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wpragmas"
```

Comment:
Is it the same thing as I did in #38431?

```
#pragma GCC diagnostic ignored "-Wunknown-pragmas"
```

Reply:
No, this one means "STFU about any warnings related to pragmas".
GCC was choking on the other ignored pragma because it didn't recognize the warning flag.